GliaWidgets/ViewModel/Chat/ChatViewModel.swift

```diff
@@ -526,16 +505,9 @@ extension ChatViewModel {
     }
 
     private func receivedMessage(_ message: CoreSdkClient.Message) {
-        guard chatStorageState().isNewMessage(message) else { return }
-
-        chatStorageState().storeMessage(
-            message,
-            interactor.queueID,
-            interactor.engagedOperator
-        )
-
         switch message.sender.type {
         case .operator:
+            guard messagesSection.items.first(where: { $0.id == message.id }) == nil else { return }
```

Comment:
I guess explanation here would be helpful.

Reply:
it is not needed anymore.